crates/holochain_state/src/source_chain/error.rs

```diff
@@ -27,6 +27,11 @@ pub enum SourceChainError {
         Option<(ActionHash, u32, Timestamp)>,
     ),
 
+    #[error(
+        "Attempted to commit a bundle to the source chain, but the CHC's head has moved since the bundle began. The source chain needs to be synced with the CHC before proceeding. Context: {0}, Original error: {1:?}"
```

Comment:
```suggestion
        "Attempted to commit a bundle to the source chain, but the CHC's head has moved since the bundle began. \
        The source chain needs to be synced with the CHC before proceeding. \
        Context: {0}, Original error: {1:?}"
```